examples/00-load/create-unstructured-surface.py

```diff
@@ -169,3 +169,66 @@
 
 # plot the grid (and suppress the camera position output)
 _ = grid.plot(show_edges=True)
+
+
+###############################################################################
+# Tetrahedral Grid
+# ~~~~~~~~~~~~~~~~
+# Here is how we can create an unstructured tetrahedral grid.
+
+# There are 10 cells here, each cell is [4, INDEX0, INDEX1, INDEX2, INDEX3]
+# where INDEX is one of the corners of the tetrahedral.
+#
+# Note that the array does not need to be shaped like this, we could have a
+# flat array, but it's easier to make out the structure of the array this way.
+cells = np.array(
+    [
+        [4, 6, 5, 8, 7],
+        [4, 7, 3, 8, 9],
+        [4, 7, 3, 1, 5],
+        [4, 9, 3, 1, 7],
+        [4, 2, 6, 5, 8],
+        [4, 2, 6, 0, 4],
+        [4, 6, 2, 0, 8],
+        [4, 5, 2, 8, 3],
+        [4, 5, 3, 8, 7],
+        [4, 2, 6, 4, 5],
+    ]
+)
+
+# 10 is just vtk.VTK_TETRA
```

Comment:
So why not use `vtk.VTK_TETRA`? It might be better practice not to add magic numbers of possible.

Reply:
This may be beside the point, but it may be a good idea to wrap these constants as variables in pyvsita. Importing vtk every time is not good for user convenience.

Reply:
If only we had an enum for this...

Reply:
Good. I will do it.